src/java.base/share/classes/java/lang/Character.java

```diff
@@ -8572,7 +8571,7 @@ public static enum UnicodeScript {
 
         private static final HashMap<String, Character.UnicodeScript> aliases;
         static {
-            aliases = new HashMap<>((int)(162 / 0.75f + 1.0f));
+            aliases = HashMap.newHashMap(162);
```

Comment:
@naotoj Seems like this magic number is likely to go out of date. Should there be a test for it like the one you updated for NUM_ENTITIES? [JDK-8283465](https://bugs.openjdk.java.net/browse/JDK-8283465).

Reply:
Good point! Filed an issue: https://bugs.openjdk.java.net/browse/JDK-8284856

Reply:
> Good point! Filed an issue: https://bugs.openjdk.java.net/browse/JDK-8284856

@stuart-marks @naotoj I can help solve JDK-8284856 after this pr. But usually we only solve 1 issue in 1 pr, so I think it's better to wait after this.

Reply:
Thanks. Will fix it myself, as I want to eliminate that immediate value in the code.

Reply:
PR opened, based on this PR.
https://github.com/openjdk/jdk/pull/8253